api/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:18
+
+# Create app directory
+WORKDIR /usr/src/api
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY package*.json ./
+
+RUN npm install
+# If you are building your code for production
+# RUN npm ci --only=production
```

Comment:
Should this be uncommented (as we're basically using for this for production from now on with the app launching soon)?

Reply:
I thought that but it didn't work, something to do with Cors. I'll have another look.

Reply:
Cool no worries, I just wondered about it based on the comment.  (Or if we don't need it could remove those two lines?)

Reply:
Might be cors is a Dev dependency when it shouldn't be? I'll check.

Reply:
It was indeed cors installed as devDependencies.